pkgs/tools/security/der-ascii/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+let
+
+pname = "der-ascii";
+version = "0.1.0";
+
+in
```

Comment:
There's no need to have this in the let block, it only breaks automatic updates.
```suggestion
```
You can use `buildGoModule rec {...` to reuse the `version`.

Reply:
I was under the impression that `mkDerivation rec { ... }` is discouraged because it breaks (IIRC) overriding? Does that not apply here or to `buildGoModule` generally, or am I just misunderstanding things?

Reply:
It is considered an "anti-pattern", but we don't have anything prohibiting from using it in Nixpkgs as far as I'm aware and since for `buildGoModule` it's the simplest solution to reuse variables. With `mkDerivation` you can do something like [this](https://github.com/NixOS/nixpkgs/blob/a89cfec0f18d84a4d5488e7bba4616ffe3dcbaf0/pkgs/development/libraries/msgpack-c/default.nix#L9).

Reply:
Speaking of overriding, `let` block most certainly will not help in that regard anyway :)

Reply:
Yes, that’s a fair point.

> With mkDerivation you can do something like [this](https://github.com/NixOS/nixpkgs/blob/a89cfec0f18d84a4d5488e7bba4616ffe3dcbaf0/pkgs/development/libraries/msgpack-c/default.nix#L9).

Whoa.